clients/clients.go

```diff
@@ -372,7 +393,7 @@ func (c *Clients) initClient(botClient *BotClient) error {
 
 	if config.AutoJoinRooms {
 		syncer.OnEventType(mevt.StateMember, func(_ mautrix.EventSource, event *mevt.Event) {
-			c.onRoomMemberEvent(client, event)
+			c.onRoomMemberEvent(botClient, event)
```

Comment:
Is there a reason for this particular change?

Reply:
So we can check the config, which is on botClient but not client